test/passing/tests/break_infix.ml.ref

```diff
@@ -112,7 +113,8 @@ let term =
       & pos 0 (some (enum modes)) None
       & info [] ~docv:"ACTION"
           ~doc:
-            (Printf.sprintf "The cache-daemon action to perform (%s)"
+            (Printf.sprintf
+               "The cache-daemon action to perform (%s)"
                (Arg.doc_alts_enum modes) ) )
```

Comment:
This is debatable. We never had a special case for `printf` kind of functions but I like having the format string on the first line.